src/functions/certGenInit.ts

```diff
@@ -12,47 +18,62 @@ import { Utils } from "../utils/Utils";
  * @param callback - callback function
  */
 const certGenInit: Handler = async (
-  event: any,
+  event: DynamoDBStreamEvent,
   context?: Context,
   callback?: Callback
-): Promise<void | Array<SendMessageCommandOutput | any>> => {
+): Promise<DynamoDBBatchResponse> => {
   if (!event) {
```

Comment:
might be worth chaning this to not event ? records  ? length or something like that and bounce it out before doing any processing if there's nothing there

Reply:
No, in my view it's still an event, it is the event that triggers the system.